cmd/main.go

```diff
@@ -198,7 +199,7 @@ func setupManager(deploymentsNamespace string, metricsAddr string, probeAddr str
 
 	clientCAName := ""
 	if enableMutualTLS {
-		clientCAName = constants.ClientCACert
+		clientCAName = filepath.Join("client-ca", constants.ClientCACert)
```

Comment:
```suggestion
                // The WebhookServer shares the same CertDir for both the server
                // certificate and the client CA certificate. We expect the ClientCACert
                // in the "client-ca"  sub-folder from the ConfigMap, since one cannot
                // mount several Secrets/ConfigMaps under the same path.
 		clientCAName = filepath.Join("client-ca", constants.ClientCACert)
```

Reply:
Adding commit, as suggested if it was needed on Slack.